digdag-core/src/main/java/io/digdag/core/database/migrate/Migration_20191105105927_AddIndexToSessions.java

```diff
@@ -0,0 +1,20 @@
+package io.digdag.core.database.migrate;
+
+import org.skife.jdbi.v2.Handle;
+
+public class Migration_20191105105927_AddIndexToSessions
+        implements Migration
+{
+    @Override
+    public void migrate(Handle handle, MigrationContext context)
+    {
+        if (context.isPostgres()) {
+            handle.update("create index concurrently sessions_on_project_id_and_workflow_name_desc on sessions (project_id, workflow_name, id DESC)");
+        } else {
+            handle.update("create index sessions_on_project_id_and_workflow_name_desc on sessions (project_id, workflow_name, id DESC)");
+        }
+    }
+
+    @Override
+    public boolean noTransaction() { return true; }
```

Comment:
please also merge https://github.com/treasure-data/digdag/pull/1248/files#diff-e9d87e9479b07f55044d89312bb8e509R22 to avoid merge conflict.

Reply:
Thank you for pointing out. I merged the PR #1248.